packages/jsapi-utils/src/TableUtils.ts

```diff
@@ -52,6 +52,7 @@ export type RowDataMapValue = {
   value: unknown;
   isExpandable: boolean;
   isGrouped: boolean;
+  visibleIndex: number;
 };
 
 export type RowDataMap = Record<string, RowDataMapValue>;
```

Comment:
We could be clearer about `RowDataMap` by using the type `ColumnName` (imported from Formatter) instead of just `string`.

```suggestion
export type RowDataMap = Record<ColumnName, RowDataMapValue>;
```